src/SFA.DAS.ApplyService.InternalApi/Services/GatewayApiChecksService.cs

```diff
@@ -82,8 +88,22 @@ public async Task<ApplyGatewayDetails> GetExternalApiCheckDetails(Guid applicati
 
         private async Task LookupCompaniesHouseDetails(ApplyGatewayDetails applyGatewayDetails, string companyNumber)
         {
-            var companyDetails = await _companiesHouseApiClient.GetCompany(companyNumber);
-            if (companyDetails == null)
+            ApiResponse<Types.CompaniesHouse.Company> companyDetails = null;
+            try
+            { 
+                companyDetails =
+                    await _retryPolicy.ExecuteAsync(context => _companiesHouseApiClient.GetCompany(companyNumber),
+                        new Context());
+
+            }
+            catch(Exception ex)
+            {
+                var message = $"Unable to retrieve results from companies house for company number {companyNumber}";
+                _logger.LogError(message, ex);
+                throw new ServiceUnavailableException(message);
+            }
+
```

Comment:
Empty line

Reply:
I;'ve taken out the empty line, and if you go to the file you can see it's been removed, so no idea why this isn't showing as 'outdated'

Reply:
Wow when I added the above comment, it now shows the fix when you view.  Fixed